deb/library50-c-6/Makefile

```diff
@@ -1,18 +1,21 @@
+src = ../../src
+
 prefix = /usr/local
-srcdir = $(prefix)/src
-libdir = $(prefix)/lib
 includedir = $(prefix)/include
+libdir = $(prefix)/lib
+srcdir = $(prefix)/src
 
-CFLAGS = --std=c99
+CFLAGS = -c -I$(src) -std=c11
```

Comment:
Let's change `c11` to `c99`. Per yesterday's thread, seems unnecessary to require compiler to support C11 when we're only using C99 features in library itself, in case others want to compile library themselves.